Sming/Libraries/CS5460/CS5460.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/CS5460.h b/CS5460.h
+index 0fbbb5b..fe7ff1d 100644
+--- a/CS5460.h
++++ b/CS5460.h
+@@ -1,7 +1,8 @@
+ #ifndef __CS5460_H__
+ #define __CS5460_H__
+ 
+-#include <arduino.h>
```

Comment:
Can you try to replace `<arduino.h>` with `<Arduino.h>` and see if the library compiles and works?

Reply:
Thanks for updating it!